### PR TITLE
Remove the border. Fix #48765

### LINKED ISSUE #48765: Pixel-perfection: Active border tab on the first tab has bad margin.
- VSCode Version: 1.22.2
- OS Version: macOS High Sierra 10.13.3

Active tab borders look great on normal tabs

screen shot 1397-02-06 at 8 41 43 pm

But the first tab has `1px` margin on the left and right sides which makes it look bad:

screen shot 1397-02-06 at 8 41 49 pm

By magnification you'll see:

screen shot 1397-02-06 at 8 44 36 pm

screen shot 1397-02-06 at 8 46 40 pm

### PR DESCRIPTION
**Pull request by shizengzhou**
@bpasero I remove the border to fix it.
border

### CODE REVIEW

File: src/vs/workbench/browser/parts/editor/media/tabstitle.css [hunk 1]
```diff
@@ -41,7 +41,6 @@
 	cursor: pointer;
 	height: 35px;
 	box-sizing: border-box;
-	border: 1px solid transparent;
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/editor/media/tabstitle.css]: @shizengzhou not a good change, this border is needed when the `tab.border` color is set, see:

image

**Review by bpasero — CHANGES_REQUESTED** [review]

**Comment by bpasero** [comment]
Maybe we should conditionally set this border if `tab.border` is defined, I think that should be possible 👍

**Comment by shizengzhou** [comment]
@bpasero I have changed my code.
tabs

**Comment by shizengzhou** [comment]
@bpasero Now I don't use borders for the separations.
tabs

**Comment by bpasero** [comment]
@shizengzhou how about keeping the border as is and letting the special case of having a border for the active tab use the `::pseudo` element trick? I actually do not like how it currently uses `box-shadow` so then we could get rid of that hack too.

**Comment by bpasero** [comment]
I found a simpler way via https://github.com/Microsoft/vscode/commit/fd7d510b0b7c72f2208910a7570ac3695e4fba53

Thanks for the work here, sorry for not pushing it 👍

**Pull request closed without merging**